microcosm_flask/tests/formatting/test_json_formatter.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test json formatting.
+
+"""
+from hamcrest import (
+    assert_that,
+    contains_inanyorder,
+    equal_to,
+    is_,
+)
+
+from microcosm.api import create_object_graph
+
+from microcosm_flask.formatting import JSONFormatter
+
+
+def test_make_response():
+    graph = create_object_graph(name="example", testing=True)
+    formatter = JSONFormatter()
+
+    with graph.app.test_request_context():
+        response = formatter(dict(foo="bar"))
+
+    assert_that(response.data, is_(equal_to(b'{\n  "foo": "bar"\n}\n')))
+    assert_that(response.content_type, is_(equal_to("application/json")))
```

Comment:
Note that the existing behavior in `jsonify` does not add an explicit `charset=utf-8` encoding to the content type. This seems weird, but isn't new, so I'm not changing it here.